### PR TITLE
Copy to clipboard

### PR DESCRIPTION
**Pull request by RoelN**
(Merge https://github.com/Wakamai-Fondue/wakamai-fondue-site/pull/112 first)

Relevant commits start at commit dc75aa1

This adds a 📋 button to all code segments so you can easily copy CSS to the clipboard.

### CODE REVIEW

File: src/components/CopyToClipboard.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	methods: {
+		copy: function() {
+			try {
+				if (!navigator.clipboard) {
+					// Use old execCommand
+					this.$refs.content.select();
+					document.execCommand("copy");
+					this.copySuccess();
+				} else {
+					// Use mew clipboard API
```
Comment (pascalw) [inline src/components/CopyToClipboard.js:18]:
```suggestion
					// Use new clipboard API
```

**Review by pascalw — APPROVED** [review]
Looks good!

One small nitpick; it would be nice if the clipboard icon would match to the padding of the textarea. No biggie, but it stood out to me as 'off'.

Screenshot 2021-02-23 at 10 03 39

File: src/components/CopyToClipboard.js (hunk 1 above)
Comment (RoelN) [reply src/components/CopyToClipboard.js:18]: 😸 Mew!

**Comment by RoelN** [comment]
@pascalw Thanks for the review! The icon is aligned technically, but since the clipboard thingy protrudes from the top, it looks slightly offset. Since this is a emoji native to the system, it will look different on Windows/Linux so I think having a technically correct margin is the best "universal" solution. (Best would be to introduce our own customs, but I'll save that for the bottom of the priority-list for now ;-))

**Pull request merged**